ReplayKitExample/ReplayKitExample/ReplayKitVideoSource.swift

```diff
@@ -0,0 +1,241 @@
+//
+//  ReplayKitVideoSource.swift
+//  ReplayKitExample
+//
+//  Created by Chris Eagleston on 9/30/18.
+//  Copyright © 2018 Twilio. All rights reserved.
+//
+
+import Accelerate
+import CoreMedia
+import CoreVideo
+import ReplayKit
+import TwilioVideo
+
+class ReplayKitVideoSource: NSObject, TVIVideoCapturer {
+
+    static let kDesiredFrameRate = 120
```

Comment:
We may want to remove the frame dropping logic entirely, because a purely timestamp based approach does not work well with ReplayKit.

Reply:
Keep it but add a comment about why it is needed, and what doesn't work as expected today.